clamav/clamav.lua

```
@@ -1,318 +1,297 @@
-function clamav:socket()
-	-- Init socket
-	local tcp_socket = socket.tcp()
-	tcp_socket:settimeout(tonumber(self.variables["CLAMAV_TIMEOUT"]))
-	local ok, err = tcp_socket:connect(self.variables["CLAMAV_HOST"], tonumber(self.variables["CLAMAV_PORT"]))
-	if not ok then
-		return false, err
-	end
-	return tcp_socket
+function clamav:api()
+  if self.ctx.bw.uri == "/clamav/ping" and self.ctx.bw.request_method == "POST" then
+    local enabled, err = has_variable("USE_CLAMAV", "yes")
```

Luacheck (1.2.0)

[warning] unused variable 'err'

(W211)